src/docs/options.md

```diff
@@ -117,6 +117,38 @@ The options available in a given Pants-enabled codebase depend on which backends
 + To see global (or goal-specific) basic options, enter `./pants help (goal)`.
 + To see global (or goal-specific) basic and advanced options, enter `./pants help-advanced (goal)`.
 
+### Compiler Options
```

Comment:
This option is zinc/javac specific, so it might be good to put in on the `jvm_projects` page?

Reply:
sure, will do!